guardian/kindle-publication-check
# Filter out spurious `getCallerClass is not supported` warning from triggering an error

## What does this change?

Explicitly filter out a warning that comes from log4j2 on java 11, that caused a "Kindle publication failed" message

## How to test

Do a test-run of kindle-gen. Then a test-run of this. You should not get a failure email.

## How can we measure success?

No spurious error messages
